test-app/tests/integration/components/bs-form-test.gjs

```diff
@@ -1314,10 +1314,10 @@ module('Integration | Component | bs-form', function (hooks) {
     );
 
     await triggerEvent('form', 'submit');
-    assert.dom('form').hasText('isSubmitted');
+    assert.dom('form').hasText('Input: isSubmitted');
```

Comment:
The `Input:` prefix makes it hard to read the test. What do you think about wrapping the assertion string in an element to have a better test selector?

Reply:
That could work. Alternatively, we could just add ignore comments for this linting rule, as I do not see much benefit for most of these cases.